lightning/log/log.go

```diff
@@ -45,7 +45,7 @@ type Config struct {
 	// Maximum number of old log files to retain.
 	FileMaxBackups int `toml:"max-backups" json:"max-backups"`
 	// Redact sensitive logs during the whole process
-	RedactLog bool `toml:"redact-log" json:"redact-log"`
+	RedactLog bool `toml:"redact-info-log" json:"redact-info-log"`
```

Comment:
We need to preserve compatibility for 5.0-rc

Reply:
5.0-rc doesn't have this feature

Reply:
if we want to be consistent with PD it should be placed inside the `[security]` section 🤔

Reply:
Sounds reasonable.